### DISCUSSION TITLE
EPEL 8 Modular going away

### DISCUSSION THREAD

**Discussion by plnordquist-pnnl** [cobbler/cobbler#3349] [General]
With EPEL 8 Modular going away as per this blog post https://fedoramagazine.org/epel-8-modularity-is-going-away/ and this EPEL issue https://pagure.io/epel/issue/198, is there going to be any way to transition from the modular rpm package in EL8 to a non modular package?  Based on the last few comments on the pagure issue it seems like they are debating whether to offer an archived version of the modular repo so I guess that might be an option depending on the outcome of that discussion.

**Comment by SchoolGuy** [comment]
@opoplawski This is your area of expertise. Are you able to help out here? I have no clue what to do.

My plan is (at minimum for openSUSE-based distros) atm to very slowly start extending <https://build.opensuse.org/project/show/home:cobbler-project> in a way that everyone has the appropriate version of Cobbler that is desired. However I have to option to invest a little bit more time and build for other distros as well in there.

**Comment by opoplawski** [comment]
My plan is to ship a plain 3.2.X cobbler package as soon as I can request an epel8 branch.  See https://pagure.io/releng/issue/11291 for the hang up there.

**Reply by plnordquist-pnnl** [reply]
If it helps the modular RPMs are listed in this repository directory https://dl.fedoraproject.org/pub/epel/8/Modular/x86_64/Packages/c/.  Not sure how they got there since it's not listed on the project overview at https://src.fedoraproject.org/rpms/cobbler.

**Reply by plnordquist-pnnl** [reply]
Oh the builds are at least listed in the build system https://koji.fedoraproject.org/koji/packageinfo?packageID=1487.
